app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -1031,6 +1031,11 @@ def build_toolbar_disable_button(id)
       when "orchestration_stack_retire_now"
         return N_("Orchestration Stack is already retired") if @record.retired == true
       end
+    when "OrchestrationTemplateCfn", "OrchestrationTemplateHot", "OrchestrationTemplateAzure", "OrchestrationTemplateVnfd", "ManageIQ::Providers::Vmware::CloudManager::OrchestrationTemplate"
+      case id
+      when "orchestration_template_remove"
+        return N_("Read-only Orchestration Template cannot be deleted") if @record.in_use?
+      end
```

Comment:
Thanks @mzazrivec, I noticed the second one but missed this one. Updated commit fixes this.